flake.nix

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Should this file not live in the `kraken-wrapper/` directory?

Reply:
It's easier for others to depend on it if it lives in the root - if it's in the root you can just write github:kraken-build/kraken-build, if it's in a subdir you have to push that path, which also means more refactoring. If we add more nixed tests for kraken, this'll be a good place to put them, too.

Reply:
Hm, the main thing that worries me is that the repository contains more than the wrapper, but the Nix flake only gives you the wrapper. 

A note on the side, can the flake be referenced by a Git tag instead? I'd like to avoid stuff referencing `HEAD` and us having to forever be backwards compatible or support the flake in that location at all.

Reply:
I think it's ok for the flake to only give you the wrapper fwiw. I think later we'd probably add some 'checks' to the nix flake, but I don't think there's any inconsistency here wrt other nix things. This is just the flexible choice.

It's definitely possible to refer to flakes by git tags. Most people select a branch because the nix flakes have a lockfile so it only updates on demand.